### PR TITLE
docs(infinite-scroll): playground examples

### PR DESCRIPTION
**Pull request by sean-perkins**
Introduces component playground examples for:
- Basic usage
- Using properties on `ion-infinite-scroll-content`
- Using custom content for the infinite scroll content

Updates the `ion-infinite-scroll-content` docs to link the usage on `ion-infinite-scroll`. 

Opted against adding playgrounds for the virtual scroll usage and for the accessibility section.
- Virtual scroll usage should really be shown with a virtual scroll library. We should consider adding support to the component playground to specify dependencies per example (will need to investigate how package-lock files effect this).
- The accessibility section doesn't have different code for the different frameworks, so showing a code snippet is the clearest way to articulate the usage. 

Quick preview:
- [`ion-infinite-scroll`](https://ionic-docs-git-fw-1288-ionic1.vercel.app/docs/api/infinite-scroll)
- [`ion-infinite-scroll-content`](https://ionic-docs-git-fw-1288-ionic1.vercel.app/docs/api/infinite-scroll-content)

### CODE REVIEW

**Comment by sean-perkins** [comment]
Should we consider any additional examples for `threshold`, `position`, etc.?

**Review by amandaejohnston — CHANGES_REQUESTED** [review]
I'm good without any additional playgrounds; I'd say the prop descriptions are self-explanatory.

File: docs/api/infinite-scroll.md [hunk 1]
```diff
@@ -23,29 +17,30 @@ import Slots from '@site/static/auto-generated/infinite-scroll/slots.md';
...
-<TOCInline
-  toc={toc}
-  maxHeadingLevel={2}
-  autogenerated={[Props, Events, Methods, Parts, CustomProps, Slots]}
-/>
+The expression assigned to the `ionInfinite` event is called when the user reaches that defined distance. When this expression has finished any and all tasks, it should call the `complete()` method on the infinite scroll instance.
 
+import Basic from '@site/static/usage/infinite-scroll/basic/index.md';
 
+<Basic />
```
[the remark this answers is not in the document]
Comment (sean-perkins) [reply docs/api/infinite-scroll.md:26]: I ended up just bumping all of them to 50. Depending on the resolution of your monitor, 30 will still be too short.

**Review by amandaejohnston — APPROVED** [review]
I'm currently seeing broken images for the avatars: https://i.gyazo.com/1823777f74c89ee86088127d870e57d8.png Not sure if it's something on my end? Everything LGTM otherwise.

**Review by liamdebeasi — CHANGES_REQUESTED** [review]

File: static/usage/infinite-scroll/custom-infinite-scroll-content/react/main_tsx.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
+```tsx
+import React, { useState, useEffect } from 'react';
+import {
+  IonContent,
+  IonInfiniteScroll,
+  IonInfiniteScrollContent,
```
[the remark this answers is not in the document]
Comment (sean-perkins) [reply static/usage/infinite-scroll/custom-infinite-scroll-content/react/main_tsx.md]: Good catch, this was just from cloning the folder for a new example. This example has a custom UI for the content, so it doesn't use the `ion-infinite-scroll-content` component.

**Comment by sean-perkins** [comment]
Working through some of the last warning tweaks, will re-ping for reviews once those are available.

**Review by liamdebeasi — APPROVED** [review]

**Pull request merged**